integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -104,7 +105,7 @@ private void deleteFsMasterJournalLogs() throws IOException {
   }
 
   private void addBlockTestUtil(FileInfo fileInfo)
-      throws IOException, InvalidPathException, FileDoesNotExistException {
+      throws IOException, InvalidPathException, FileDoesNotExistException, AccessControlException {
```

Comment:
I think that we do not need to enumerate all the exceptions a test can throw ... simple catch Exception.

This applies to all functions in this file.